src/shared/components/mutationTable/column/clonal/ClonalColumnFormatter.tsx

```diff
@@ -9,17 +9,20 @@ import ClonalElement from 'shared/components/mutationTable/column/clonal/ClonalE
  */
 
 export enum ClonalValue {
-    YES = 'yes',
-    NO = 'no',
+    CLONAL = 'clonal',
+    SUBCLONAL = 'subclonal',
+    INDETERMINATE = 'indeterminate',
     NA = 'NA',
 }
 
 function getClonalValue(mutation: Mutation): ClonalValue {
     let textValue: ClonalValue = ClonalValue.NA;
     if (hasASCNProperty(mutation, 'clonal')) {
-        textValue = mutation.alleleSpecificCopyNumber.clonal
-            ? ClonalValue.YES
-            : ClonalValue.NO;
+        textValue =
+            mutation.alleleSpecificCopyNumber.clonal in ClonalValue &&
+            mutation.alleleSpecificCopyNumber.clonal !== 'INDETERMINATE'
```

Comment:
why the comparison is not to the enum ClonalValue?

Reply:
backend response is coming back with uppercase value but I can compare it to the key.

Reply:
Can the clonal returned from backend be a Enum? If that's too much to ask, then you can make your Enum in the frontend as all uppercase, then do a force type (mutation.alleleSpecificCopyNumber.clonal as ClonalValue). The lowercase value is useful at the rendering. You can do all comparison logics in uppercase and at the place renders the content, do a .toLowerCase?